index.bs

```diff
@@ -274,11 +276,11 @@ In general, it is recommended to use properties instead of [=Metadata OBUs=] in
 
 [[!AV1]] supports encoding a frame using multiple spatial layers. A spatial layer may improve the resolution or quality of the image decoded based on one or more of the previous layers. A layer may also provide an image that does not depend on the previous layers. Additionally, not all layers are expected to produce an image meant to be rendered. Some decoded images may be used only as intermediate decodes. Finally, layers are grouped into one or more [=Operating Points=]. The [=Sequence Header OBU=] defines the list of [=Operating Points=], provides required decoding capabilities, and indicates which layers form each [=Operating Point=].
 
-[[!AV1]] delegates the selection of which [=Operating Point=] to process to the application, by means of a function called <code>choose_operating_point()</code>. AVIF defines the <code>[=OperatingPointSelectorProperty=]</code> to control this selection. In the absence of an <code>[=OperatingPointSelectorProperty=]</code> associated with an [=AV1 Image Item=], the AVIF renderer is free to process any [=Operating Point=] present in the [=AV1 Image Item Data=]. In particular, <assert>when the [=AV1 Image Item=] is composed of a unique [=Operating Point=], the <code>[=OperatingPointSelectorProperty=]</code> should not be present</assert>. If an <code>[=OperatingPointSelectorProperty=]</code> is associated with an [=AV1 Image Item=], the <code>[=op_index=]</code> field indicates which [=Operating Point=] is expected to be processed for this item.
+[[!AV1]] delegates the selection of which [=Operating Point=] to process to the application, by means of a function called <code>choose_operating_point()</code>. [=/AVIF=] defines the <code>[=OperatingPointSelectorProperty=]</code> to control this selection. In the absence of an <code>[=OperatingPointSelectorProperty=]</code> associated with an [=AV1 Image Item=], the [=/AVIF=] renderer is free to process any [=Operating Point=] present in the [=AV1 Image Item Data=]. In particular, <assert>when the [=AV1 Image Item Data=] is composed of a unique [=Operating Point=], the <code>[=OperatingPointSelectorProperty=]</code> should not be present</assert>. If an <code>[=OperatingPointSelectorProperty=]</code> is associated with an [=AV1 Image Item=], the <code>[=op_index=]</code> field indicates which [=Operating Point=] is expected to be processed for this item.
```

Comment:
Regarding `AV1 Image Item` => `AV1 Image Item Data`, see issue #220.
In my mind, I see it like this:
1. The AV1 Image Item Data may contain one operating point or multiple operating points
2. The AV1 Image Item **always** consists of a unique operating point since the `a1op` defaults to selecting operating point 0 if not present.

So making statements about "when the AV1 Image Item is composed of a unique operating point" feels like something that is always true, but the text then says that if so the property should not be present.

Happy to revert if people disagree.

Reply:
Hmmm... Good catch. Let me think while I write.

Section 2.2.2 says:
- No `lsel` or `lsel == 0xFFFF`
    - `a1op` missing => `ispe` maps to last frame in operating point 0
    - `a1op` present => `ispe` maps to last frame in operating point specified by `a1op`
- `lsel != 0xFFFF`
    - `a1op` missing => `ispe` maps to frame selected by `lsel` (operating point not actually specified)
    - `a1op` present => `ispe` maps to frame selected by `lsel` **in operating point specified by `a1op`**

For the case of `lsel == 0xFFFF`, the renderer is free to choose intermediate frames for progressive decoding, but it's fully defined what is considered the final output image. If `a1op` is present, only the layers in that operating point are used. **If `a10p` is not present, I think the renderer is technically allowed to choose any operating point as long as the final frame is the last frame in operating point 0.**

For the case of `lsel` being present and selecting an actual layer, I guess **the renderer is technically allowed to select any operating point that contains the layer if there is no `a1op` property.**

For the case of `lsel` not being present, the AV1 data is not supposed to be multilayered. **I think this means you can't really have multiple operating points.**

-----------

To summarize, I think I now understand why @cconcolato considered both the old and the new text equivalent. You **can** have an image item that has an unspecified operating point. The final output layer is fully defined, but that layer may belong to multiple operating points, and the writer does not technically need to specify which to use.

Given this I think I'll revert this change.